testsuite/features/secondary/srv_power_management_redfish.feature

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) 2020 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Power management
+
+  Scenario: Fake a Redfish host
```

Comment:
I would just hide the point that we are moking it, in our Gherkin scenarios.
The implementation we make behind the scenarios shouldn't be important at the test description level

Reply:
done